frontend/src/metabase/collections/components/BaseTableItem.jsx

```diff
@@ -54,7 +54,6 @@ export function BaseTableItem({
   onMove,
   onDrop,
   onToggleSelected,
-  ...props
```

Comment:
Not sure why this unused var isn't caught by our linter since this is a javascript file